server/server.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Main application file
+ */
+
+'use strict';
+
+// Set default node environment to development
+// process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+var express = require('express');
```

Comment:
put ALL requires in all node files at the top of the file in this order
- var nodemodule = require('nodemodule')
- var localfile = require('localfile.js')